src/unity/python/turicreate/visualization/show.py

```diff
@@ -7,6 +7,17 @@
 from __future__ import division as _
 from __future__ import absolute_import as _
 
+def get_client_app_path():
```

Comment:
Since this function is only used internally within turicreate (and users shouldn't call it directly), let's prefix the name with an underscore so it doesn't show up in tab completion: `_get_client_app_path`.